cmake/std/atdm/contributed/blake/environment.sh

```diff
@@ -97,48 +106,47 @@ else
   export OMP_NUM_THREADS=1
 fi
 
-if [[ "$ATDM_CONFIG_COMPILER" == "INTEL-18.1.163" ]] ; then
-  module load devpack/20171203/openmpi/2.1.2/intel/18.1.163
-  module swap cmake/3.9.0 cmake/3.19.3
-  module load ninja/1.7.2
-  module load python/2.7.13
-
-  export LOCAL_ARCH_FLAG="-xCORE-AVX512 -mkl"
-  export LOCAL_BLAS_LIBRARIES="-mkl;${MKLROOT}/lib/intel64/libmkl_intel_lp64.a;${MKLROOT}/lib/intel64/libmkl_intel_thread.a;${MKLROOT}/lib/intel64/libmkl_core.a"
-  export LOCAL_LAPACK_LIBRARIES=${LOCAL_BLAS_LIBRARIES}
+if [[ "$ATDM_CONFIG_COMPILER" == "INTEL-2021.4.0" ]] ; then
+  module load intel/oneAPI/hpc-toolkit/2021.4.0 intel/oneAPI/base-toolkit/2021.4.0 
+  module load openmpi/4.0.5/intel-oneapi/2021.4.0 
+  module swap gcc/7.2.0 gcc/10.2.0
+  module load openblas/0.3.21/gcc/10.2.0
+  module load boost/1.75.0/intel-oneapi/2021.2.0
+  module load hdf5/1.10.7/openmpi/4.0.5/intel-oneapi/2021.2.0 netcdf-c/4.7.4/openmpi/4.0.5/intel-oneapi/2021.2.0 zlib/1.2.11
+  module load python/3.7.3
+  #module load python/2.7.13
```

Comment:
Has this been tested on blake? Might try to use this for Albany.

Reply:
@jewatkins I've been running the gnu/11 option in a nightly build (OpenMP+Serial, debug build, SKX arch) using the [cmake/std/atdm/contributed/blake/environment.sh](https://github.com/trilinos/Trilinos/pull/11988/files/b36484ff1af4937557a3aa160afafea3af5ae849#diff-8e7cd8c90aa32e48f27bad72361a472bb53331c125743e66b3a413f718c89886) script

I've been testing with the intel/2021.4 changes but have run into issues with netcdf when trying to enable SEACAS and have not worked that out yet (not sure if module/system related or user error)

Reply:
@jewatkins also, lots of failing unit tests in OpenMP builds with intel/2021.4 (various issues opened)

Reply:
Ah okay, we've run into similar issues. We decided to just drop our intel build and use the gcc build. I mentioned the issues with intel modules to admins but I haven't heard of anything new coming.